docs/_data/buttons.json

```diff
@@ -35,6 +35,26 @@
         }
       ]
     }
+  ]
+  ,
+  "featured": [
```

Comment:
I went with `featured` for this variant name. 

I figured it's best to avoid coupling naming with visual that could change (`purple`) and `new` felt too restrictive since I could see this button being used just to grab some attention generally. `featured` felt like the best fit but I'm glad to change it if you feel differently @CGuindon.